src/sharedInterfaces/queryResult.ts

```diff
@@ -27,12 +66,37 @@ export interface QueryResultWebviewState {
     resultSetSummaries: { [key: number]: ResultSetSummary };
     messages: IMessage[];
     tabStates?: QueryResultTabStates;
+    isExecutionPlan?: boolean;
+    executionPlanState: {
```

Comment:
Why not make an interface out of it and share it with the regular plan webview. Let's try avoid duplicate types since they tend to get out of sync with each other.